scripts/coverage_report.sh

```diff
@@ -81,7 +81,7 @@ echo "Cleaning project..."
 echo "Running tests..."
 while read -r line; do
         dirline=$(realpath $(dirname "$line"));
-        (cd "$dirline"; cargo +nightly test --all-features)
+        (cd "$dirline"; cargo +nightly xtest)
```

Comment:
Yeah this is the right thing to do because just running with `--all-features` would break in the crypto crate.